mtp_noms_ops/apps/security/utils.py

```diff
@@ -38,7 +38,6 @@ def convert_date_fields(object_list, include_nested=False):
     supported yet.
     """
     fields = ('started_at', 'received_at', 'credited_at', 'refunded_at', 'created', 'triggered_at', 'actioned_at')
-    parsers = (parse_datetime, parse_date)
```

Comment:
@ushkarev I've checked these fields and they all seem to be datetimes, so I've removed the logic that tried to parse `date`s.

The change to [Django 4.x' `parse_datetime()`](https://code.djangoproject.com/ticket/35097) shouldn't matter now (unless I'm missing something) and this logic _**should**_ work both on currently used Django 3.2 and on Django 4.0.

Reply:
(but of course have a look and see if this makes sense to you as as well, especially as datetimes can be sneaky)